orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/restore_db.sh

```diff
@@ -1,47 +1,37 @@
 #!/bin/bash
 # restore_db.sh
 # This script uses the 'psql' cli to drop hive and ranger databases, create them, then read in a plain SQL file to restore data.
-# We use the `azcopy copy` and `aws s3 cp` commands to copy backups from Azure or AWS respectively.
-# When using either cloud provider, the plaintext SQL files are first copied to the local filesystem, then fed into psql.
+# We retrieve the SQL files from a valid HDFS URL, which should be an `s3a://` or `abfs://` for AWS or Azure clouds respectively.
```

Comment:
URL is not an HDFS URL . You can just say " We retrieve the SQL files from a valid URL, ..."

Reply:
Done